scripts/env.sh

```diff
@@ -27,6 +27,11 @@ WORKDIR=/opt/cita-run
 USER_ID=`id -u $USER`
 USER_NAME="user"
 
+if [[ "${1}" =~ "create_cita_config" ]]; then
```

Comment:
Why use `=~` ?

Reply:
For the future that If we refactor the create_cita_config.

More detail:
I want to use the same `env.sh`, so i need to differentiate two image type.
if `./env.sh *create_cita_config* xx`, then will use cita-config image.